pyaccel/new_lifetime.py

```diff
@@ -49,10 +53,13 @@ def lifetime(model, Vx, Vy, spread, P, I, n_bunches, emit, sig_S, acoplamento, a
     By       = twiss.betay
     Ib = I/n_bunches
     T_rev    = pyaccel.optics.get_revolution_period(model)#s
-    sig      = pyaccel.optics.get_beam_size(model)
     eta_x    = twiss.etax
+    length   = pyaccel.lattice.get_attribute(model, 'length')
+    L        = sum(length)
+    Vac_x    = np.array(pyaccel.lattice.get_attribute(model, 'vmax'))-np.array(pyaccel.lattice.get_attribute(model, 'vmin'))
```

Comment:
@AnaLuisaAR, is this correct? I think what is used in the lifetime calculation is the half aperture of the chamber, not the whole aperture. If this is the case, I think it is more appropriated to take the minimum of the negative and positive apertures. What do you think?